frontend/src/app/shared/box-table/box-table.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="boxTable">
+  <div class="oib-box-title d-flex">
+    <div class="flex-grow-1 title-container">
+      <ng-container *ngIf="titleTemplateRef; else stringTitle">
+        <ng-container *ngTemplateOutlet="titleTemplateRef"></ng-container>
+      </ng-container>
+      <ng-template #stringTitle>{{ boxTitle | translate }}</ng-template>
+    </div>
+  </div>
+  <div class="oib-table-content">
+    <ng-content></ng-content>
+  </div>
+</div>
```

Comment:
This component does basically the same thing than oib-box. This should be removed and use oib-box instead